examples/connector_templates/README.md

```diff
@@ -0,0 +1,25 @@
+# Using templates to implement Secretless Connector Plugins
+
+We created connector templates to ease the process of adding new connectors to secretless.
+Before using the templates to add new connector plugins, be sure to read the [Secretless Connector Plugins README](https://github.com/cyberark/secretless-broker/blob/master/pkg/secretless/plugin/connector/README.md)
+
+To add a new connector do the following:
+
+1. Copy the relevant template directory (HTTP/TCP) into `internal/plugin/connectors/<connector type>`. 
```

Comment:
how do you know which one to copy? is it worth referring to [the connector docs](https://github.com/cyberark/secretless-broker/tree/master/pkg/secretless/plugin/connector#technical-overview) to learn more if you're not sure which one you need to copy?

Reply:
@izgeri is this line ok?

```
If you're not sure which connector type is suitable, please refer to the [connector technical overview](https://github.com/cyberark/secretless-broker/tree/master/pkg/secretless/plugin/connector#technical-overview)
```